src/Stratis.Bitcoin.IntegrationTests/Wallet/WalletAddressGenerationAndFundsVisibility_Steps.cs

```diff
@@ -38,6 +39,9 @@ protected override void BeforeTest()
 
         protected override void AfterTest()
         {
+            this.sendingStratisBitcoinNode.FullNode.Network.Consensus.CoinbaseMaturity = this.previousCoinBaseMaturity;
```

Comment:
Changes in WalletAddressGenerationAndFundsVisibility_Steps.cs do not belong to this PR

Reply:
OK, you are right, however I am just trying to make easy to understand PRs and not waste too much time in the process. This is an important fix too because not restoring CoinBaseMaturity leads to the value being changed for all subsequent tests, and test failures. It is easy to understand this very local change...

I should rename the PR to the branch it is based on : integration-test-small-fix :)

Reply:
Ah, so that's fixing integration tests that are currently failing on master?